data/korath/korath missions.txt

```diff
@@ -257,3 +257,331 @@ mission "Small Scale Delivery"
+			`	Fortunately, there are plenty of Humans who are willing to risk it for the right pay, even given the comfortable life they have in Hai space. You show the new crew members to their bunks and prepare to salvage the <EDC-ship>.`
+
+ship "Charm-Shallop" "Charm-Shallop (Distressed)"
+	outfits
+		"Digger Mining Beam" 2
+		"Fuel Processor"
+		"Generator (Furnace Class)"
+		"Large Heat Shunt"
+		"Plasma Core"
+		"Scram Drive"
+		"Small Heat Shunt"
+		"Steering (Planetary Class)"
+		"Thermal Repeater Rifle" 19
+		"Thruster (Lunar Class)"
+		"Warder Anti-Missile"
+	turret 0 -35 "Warder Anti-Missile"
+	turret 0 97
+	gun 0 -118 "Digger Mining Beam"
+	gun -16.5 -114.5 "Digger Mining Beam"
+	gun 16.5 -114.5
+
+mission "Efreti: Distress Call 6"
+	name "Salvage the <EDC-ship>"
+	description "Salvage the <EDC-ship> and return it to <planet> to claim rights to it."
+	source "Greenwater"
+	destination "Seleptra Nak"
+	landing
+	passengers 66
+	blocked "You need to be able to carry <bunks> additional crew to salvage the ship. Return here when you have sufficient crew capacity."
+
+	to offer
+		has "Efreti: Distress Call 5: done"
+
+
+mission "Efreti: Distress Call 6 Ships"
+	invisible
+	landing
+	source "Greenwater"
+	to offer
+		has "Efreti: Distress Call 5: done"
+	to fail
+		or
+			has "Efreti: Distress Call 7: failed"
+			has "Efreti: Distress Call 7: aborted"
+			has "Efreti: Distress Call 7: done"
+	npc kill
+		system "Seketra"
+		fleet "Small Kor Mereti" 2
+		government "Kor Mereti"
+		personality
+			staying
+
+
+mission "Efreti: Distress Call 7"
+	landing
+	name "Salvage the <EDC-ship>"
+	description "Salvage the <EDC-ship> and return it to <planet> to claim rights to it."
+	source "Seleptra Nak"
+	destination "Karek Fornati"
+	landing
+	to offer
+		has "Efreti: Distress Call 6: done"
+	on offer
+		conversation
+			`Thanks to the work of your crew, the atmosphere on the ship is brought to human standards, if a bit tropical. As the salvage continues, you and your crew start to become familiar with the ship. Its strong hull appears spaceworthy, but the ship has barely any weapons, the battery systems are beyond repair, and shield regeneration is impossible. If the Mereti attack, the <EDC-ship> won't be able to survive long.`
+			choice
+				`	(Fly the <EDC-ship> home.)`
+					goto runforit
+				`	(Abandon the ship.)`
+					goto abandon
+			
+			label abandon
+			action
+				fail
+			`	Your crew abandons the Korath ship to the elements, just one more tragic story amid the ruins of their civilization.`
+				flee
+			
+			label runforit
+			`	You manage to bring up the shields, at least initially, using the <ship>. Your crew on the <EDC-ship> take their stations as you prepare to evade Mereti patrols on your way to <destination>.`
+				launch
+
+	on complete
+		fail "Efreti: Distress Call 6 Ships"
```

Comment:
If you're going to fail it here, don't put the auto condition set by this action (`[...]: done`) in the `to fail` of that mission. It's just confusing for anyone reading it.

Reply:
I will try removing this one and see if it works.

Reply:
It works.